assets/css/content/admonition.css

```diff
@@ -1,36 +1,36 @@
-.content-inner blockquote:is(.warning, .error, .info, .neutral, .tip) {
+.content-inner section.admonition:is(.warning, .error, .info, .neutral, .tip) {
```

Comment:
We can now simplify most of these rules. We no longer need `:is(...)`, since the section.admonition is enough to identify it.

Reply:
I don't have any objections to that, but it seems like the `:is` actually simplifies most of the selectors here. That is, removing `:is` for `.content-inner section.admonition:is(.warning, .error, .info, .neutral, .tip)` would result in

```css
.content-inner section.admonition.warning,
.content-inner section.admonition.error,
.content-inner section.admonition.info ...
```

Do you still want me to move forward with "de-`:is`ing"?

Reply:
@voughtdq, I think that for the rules that select all of the types, we can simplify to `.content-inner .admonition`?

Reply:
Fixed that. I'm going to look closer at the selectors that use `:is` and see if some of them can be simplified.

Reply:
Okay. To clarify, whereas before we needed to include all the types of admonitions because there could be other types of blockquotes on the page, now that there is a section with a class of `admonition` I think selecting using that single class should be possible. Does that make sense?

Reply:
Yeah, to be clear, this should simply be: `content-inner section.admonition {`

Reply:
I would keep the section to avoid accidentally applying styles. Sorry for the confusion.